games-strategy/ufo-ai/ufo-ai-2.5.ebuild

```diff
@@ -0,0 +1,130 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=3
+inherit eutils flag-o-matic games
+
+MY_P=${P/o-a/oa}
+
+DESCRIPTION="UFO: Alien Invasion - X-COM inspired strategy game"
+HOMEPAGE="http://ufoai.sourceforge.net/"
+SRC_URI="mirror://sourceforge/ufoai/${MY_P}-source.tar.bz2
+	mirror://sourceforge/ufoai/${MY_P}-data.tar"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+IUSE="debug server editor +client sse profile"
+
+# Dependencies and more instructions can be found here:
+# http://ufoai.ninex.info/wiki/index.php/Compile_for_Linux
+DEPEND="!server? (
+		virtual/opengl
+		 virtual/glu
+		media-libs/libsdl2
```

Comment:
are there _no_ USE flags at all necessary on libsdl2? as in... if you compile `media-libs/libsdl2` with __all__ USE flags disabled, does ufo-ai still work?